features/kinderen.feature

```diff
@@ -77,84 +86,86 @@ Functionaliteit: Kinderen van een persoon raadplegen
-      | 9         |                             |                   |                     | <geslachtsnaam>       | <geboortedatum>       | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+    Scenario: kind volledig onbekend, veld met onbekend waarde wordt gevraagd met fields
+      Gegeven de persoon met burgerservicenummer '000003456' heeft een 'kind' met de volgende gegevens
+      | geslachtsnaam (02.40) | 
+      | .                     |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999996150                       |
+      | burgerservicenummer | 000003456                       |
       | fields              | kinderen.naam                   |
-      Dan heeft de persoon met burgerservicenummer '999996150' een kind met de volgende 'naam' gegevens
-      | naam          | waarde          |
-      | geslachtsnaam | <geslachtsnaam> |
-
-      Voorbeelden:
-      | geslachtsnaam | geboortedatum |
-      | .             |               |
-      | .             | 00000000      |
+      Dan heeft de response een persoon met een 'kind' met de volgende gegevens
+      | naam               | waarde |
+      | type               | Kind   |
+      | naam.geslachtsnaam | .      |
 
-      @gba
-      Scenario: kind volledig onbekend, veld met onbekend waarde wordt niet gevraagd met fields
-        Gegeven de persoon met burgerservicenummer 999996150 heeft de volgende kinderen in de registratie
-        | Categorie | Burgerservicenummer (01.20) | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
-        | 9         |                             |                   |                     |                       | 00000000              | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
-        Als personen wordt gezocht met de volgende parameters
-        | naam                | waarde                          |
-        | type                | RaadpleegMetBurgerservicenummer |
-        | burgerservicenummer | 999996150                       |
-        | fields              | kinderen.naam                   |
-        Dan heeft de persoon met burgerservicenummer '999996150' een kind zonder gegevens
+    @gba
+    Scenario: kind volledig onbekend, veld met onbekend waarde wordt niet gevraagd met fields
```

Comment:
Zoals het nu is gespecificeerd heeft een kind helemaal geen naam gegevens. In de oude versie heeft het kind onbekend geslachtsnaam en geboortedatum. In de nieuwe scenario, zou de Dan stap het volgende moeten zijn: `Dan heeft de response een persoon met een 'kind' zonder gegevens`. In de oude versie zou de Dan stap zijn: `Dan heeft de response een persoon met een 'kind' zonder 'naam' gegevens`

Reply:
Als ik 'Dan heeft de response een persoon met een 'kind' zonder gegevens' gebruik dan is de response van de proxy: 

`[ ]`

Daar moet volgens mij dus ook iets aangepast worden.

Reply:
> Als ik 'Dan heeft de response een persoon met een 'kind' zonder gegevens' gebruik dan is de response van de proxy:
> 
> `[ ]`
> 
> Daar moet volgens mij dus ook iets aangepast worden.

Dat kan kloppen. Voor de vakantie was ik bezig met de implementatie, maar dat is nog niet af

Reply:
ik zou het ook duidelijker vinden als voor dit scenario en het vorige scenario de Gegeven stappen identiek zouden zijn. Dan zie je bij dezelfde gegevens het effect van andere fields.

Dus wat mij betreft alleen wijzigen wat je expliciet wilt testen/tonen en de rest ongewijzigd laten.

Je zou dus in beide scenario's zowel geboortedatum "00000000" en geslachtsnaam "." kunnen opgeven. In het eerste scenario vraag je kinderen.naam, in het tweede vraag je bijvoorbeeld kinderen.burgerservicenummer.

Reply:
Is aangebracht conform wens.